src/client/README.md

```diff
@@ -109,6 +109,20 @@ npm run storybook
 _Tech Note: this is running the standard react storybook local server (which is based on [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware)) with a configured public path of /storybook, for deployment purposes.
 This means that we need a customer middleware.js to redirect, therefore the landing page resolves to http://localhost:6006/storybook start url_
 
+There seems to be some issues with storybook cache on some machines, there are 2 remedies:
+
+1. Add --no-manager-cache to the storybook script
```

Comment:
since this is already in the script, this line seems a bit confusing. perhaps if we
1. update this line to say "run `npm run storybook -- --no-manager-cache`", and
2. remove the `--no-manager-cache` from the npm script in package.json
that would be an improvement?

Reply:
I think this is universally required, until further notice.
As we are not running SB every day, I don't think it will hurt us much.
Also, I have (some) hopes that running vite builder will be better, as long as we can configure the public /storybook address successfully

Reply:
I appear to have committed the script change, so not much needed in here now, just a short explanation and link to the SB CLI page 😄